README.md

```diff
@@ -125,6 +140,14 @@ android update sdk --filter tools,platform-tools,android-23,build-tools-23.0.2,s
 
 > On OS X systems, you can develop, build, and deploy NativeScript projects that target iOS and Android.
 
+### Setup Script
+To quickly set up your system for the latest NativeScript CLI, run the following Ruby setup script in the **Terminal**:
```

Comment:
Why not go with the "paste the following x in y and hit Enter" phrasing from above?